app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -34,4 +34,8 @@
   <%= render "govuk_publishing_components/components/inset_text" do %>
     <p class="govuk-body">Not yet claimed for 2i</p>
   <% end %>
+<% elsif @step_by_step_page.status.in_review? %>
+    <%= render "govuk_publishing_components/components/inset_text" do %>
+      <p class="govuk-body">Step by steps cannot be published until it is 2i approved.</p>
```

Comment:
Could I propose:
```
Step by step cannot be published until it is 2i approved.
```

Reply:
I changed it in line with Alex and Chris' suggestion (https://gds.slack.com/archives/CLJCUGVD2/p1569511783011000) and updated the screenshots.